ocs_ci/ocs/resources/pod.py

```diff
@@ -498,6 +498,23 @@ def get_csi_provisioner_pod(interface):
     return provisioner_pod
 
 
+def get_csi_snapshotner_pod():
+    """
+    Get the csi snapshot controller pod
+
+    Returns:
+        Pod object: csi snapshot controller pod
+
+    """
+    ocp_pod_obj = OCP(
+        kind=constants.POD, namespace='openshift-cluster-storage-operator'
+    )
+    selector = 'app=csi-snapshot-controller'
+    snapshotner_pod = ocp_pod_obj.get(selector=selector)['items']
+    snapshotner_pod = (Pod(**snapshotner_pod[0]).name)
```

Comment:
I believe there are 2 pods - one active and one standby? how do we know if we are looking at the right pod?

Reply:
For the controller there is only one, not like the provisioner

Reply:
Yes it currently only one pod which is deployed as statuefulset, (this will be deployed by the OCP team)